speed-dreams/source-2.2.3/data/data/menu/DeveloperMenu.xml

```diff
@@ -12,11 +12,11 @@
             <section name="options">
                 <section name="0">
                     <attstr name="text" val="Synchronous Black Box"/>
-                    <attstr name="tip" val="Select for the simulation to wait on the black box"/>
+                    <attstr name="tip" val="Select for the simulation to wait on the black box interventions"/>
```

Comment:
```suggestion
                    <attstr name="tip" val="Select for the simulation to wait for the black box interventions"/>
```
To wait **on** means the simulation will wait **when** it receives a reply instead of **until**.